analyze.module

```diff
@@ -25,4 +29,232 @@ function analyze_theme() {
       ],
     ],
   ];
-}
\ No newline at end of file
+}
+
+/**
+ * Implements hook_menu_local_actions_alter().
+ */
+function analyze_menu_local_actions_alter(&$local_actions): void {
+
+  /** @var \Drupal\analyze\AnalyzePluginManager $service */
+  $service = \Drupal::service('plugin.manager.analyze');
+
+  // Add local actions for all entities if we have plugins enabled.
```

Comment:
I think we discussed having plugins enabled at a per entity/bundle level? That could follow as an enhancement though.

Reply:
Do we mean these to be actions (e.g. "Add") rather than 2nd level tasks?

Reply:
> Do we mean these to be actions (e.g. "Add") rather than 2nd level tasks?

These currently appear below the default entity tabs with the "analyze" tab as their parent: was there another way you expected them to appear?

Reply:
Yup, access rather than discovery makes a lot of sense :+1:

Reply:
![image](https://github.com/user-attachments/assets/92976e46-c222-4dac-b1df-73f66f4989dc)
So the links above are _actions_ (which matches the code).

What I think you should be using are 2 levels of task, which will looks like this:
![image](https://github.com/user-attachments/assets/b16f2503-1aef-4638-9909-51f9dcad5de9)

Reply:
This is fixed now, however since we only have one plugin that gives back a page it will not show up unless there are two or unless we set a default.

Here is the result with spoofing that the node views has a page as well:
![image](https://github.com/user-attachments/assets/b3c921cd-9211-4a6b-a256-937f68201e7f)